practice/src/css/bases/variable.css

```diff
@@ -29,7 +29,7 @@
   --font-size-3xl: 36px;
   --font-size-bg: 52px;
 
-  --primary-background-color: #fff;
+  --primary-background-color: #ffffff;
```

Comment:
Should use shorthand. You can refer this article  https://www.freecodecamp.org/news/how-hex-code-colors-work-how-to-choose-colors-without-a-color-picker/